### DISCUSSION TITLE
’Species Stats‘ no longer show anything

### DISCUSSION THREAD

**Discussion by FirstTenor** [mcguirepr89/BirdNET-Pi#735] [Q&A] [answered]
I have been running birdnet-pi for about a week - it reports about 4000 detections - but I have plenty of memory left (128Gb so card on a P-Pi 4B 4Gb).  I regularly check for upgrades and install those (Tools > System controls > Update).   Since the last update a couple of days ago, I no longer see any ‘Species Stats’ …. It just shows a blank screen.  I left it running for a while - but nothing ever displays.  I have also tried this os several browsers & terminals.

I have also stopped and restarted the services a few times.

Any suggestions as to how to re-enable the statistics display …. That should start to get interesting now that I have built up a few days of logs and I was hoping to start to see a pattern for some species.

I have looked in discussions 
![5EC5759E-5F2D-46CB-ACA8-6E3D5B1C875B](https://user-images.githubusercontent.com/7486381/217941701-863f2c63-72ee-4020-9a80-aa26b6101adc.jpeg)
and not found any similar questions - so dont know if this is just me or if others have seen this

**Comment by ehpersonal38** [comment, marked as answer]
Hi, thanks for searching the discussions before posting this!

I would recommend these actions for diagnosing this:
1. Check if `birdnetpi.local/stats` displays anything other than a blank page
2. Check the output of this command: `sudo service birdnet_stats status`. Please attach a screenshot of this output so I can get some more information as to what may be going awry here

**Comment by FirstTenor** [comment]
Thanks for response & diagnosis suggestion.

I’ll attach both screenshots - it may help others
Direct view of the stats showed a blank page

![F57A6894-15AD-4629-A5F9-95FBDF0B789C](https://user-images.githubusercontent.com/7486381/218038954-b7bdcd8d-0861-4e2d-a219-205581890152.jpeg)

So I ran the command in a separate ssh login …. 

![80FD1A4F-16AC-4365-967C-6787E4DDE5C3](https://user-images.githubusercontent.com/7486381/218039375-fc2ca86c-da6d-4fda-9c64-f601b84d6ce1.jpeg)

Now going back to the stats page shows the stats

![AA87D12C-DF58-4866-98E0-7DBC7CA82656](https://user-images.githubusercontent.com/7486381/218040307-be5fd77e-0b07-4caa-af9e-70066f0d7942.jpeg)

 … so I think that this is now answered.

**Reply by ehpersonal38** [reply]
I mean I'm glad to see that you're back up and running, but that command shouldn't have done anything to fix it? All that command does is basically just print out the logs.

**Reply by FirstTenor** [reply]
Ok - thanks, I will keep an eye on it.  The stats do seem to be working again…. I had expected that command to print out some info… I am not sure if I looked at the stats since the rebuild that I needed to run last week (an OS upgrade failed and I burnt a new image to restart).  I guess the first stats page could have been damaged and that command re-evaluated it ????  …. Anyway - thanks for the suggestion :-)
